cmd/pd-tso-bench/main.go

```diff
@@ -82,6 +82,9 @@ func main() {
 func showStats(ctx context.Context, durCh chan time.Duration) {
 	defer wg.Done()
 
+	statCtx, cancel := context.WithCancel(ctx)
+	defer cancel()
```

Comment:
The cancel function is for the statCtx resource, it is better to do it even it may has no effect.